### ISSUE TITLE
T1 header differences on new PNS scans

### ISSUE THREAD

**Issue by navonacalarco** [TIGRLab/admin#865]
![screen shot 2016-11-17 at 11 36 10 am](https://cloud.githubusercontent.com/assets/13733654/20398120/1173581a-acba-11e6-83cb-b6e68d960a7b.png)

investigate importance, effects Ps 23 and 25.

**Comment by josephdviviano** [comment]
Hey @navonacalarco -- these look meaningful, good catch. I wouldn't blacklist these scans (they aren't different enough to really impact the data), but the MRI techs should fix these issues.

**Comment by navonacalarco** [comment]
Interestingly, 3/4 of the PNS participants with a variety of header differences are shared with SPINS (i.e., had a single scan session, but participated in both the PNS and SPINS study via datasharing). The acquisitions between SPINS and PNS are supposed to be identical, but the corresponding SPINS QC pages do _not_ have a header difference. 

I looked at the gold standard for PNS (based on the PNS_CMH_0003) and everything looks ok there, so I'm kind of at a loss.

I'm deleting the QC pages just in case the gold standard for PNS was added/ changed after the pages were made.
